11/jre/slim-aws/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM openjdk:11.0.1-jre-slim
+MAINTAINER Forest Keepers
+
+RUN apt-get update && \
+    apt-get install -y \
+      curl \
+      jq \
+      openssl \
+      net-tools \
+      python-pip \
+      wget && \
+    rm -rf /var/lib/apt/lists/*
+
+ENV AWSCLI_VERSION 1.16.47
+
+RUN pip install awscli==$AWSCLI_VERSION
+
+RUN echo 'source $HOME/aws/env/bin/activate' >> .bashrc && \
+    echo 'complete -C aws_completer aws' >> .bashrc
```

Comment:
This images are for runtime, to host a java service. No reason to have bash completion here.

Reply:
Removed bash completion